core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -2352,7 +2352,7 @@ class ReplicaManagerTest {
     val (rm0, rm1) = prepareDifferentReplicaManagers(mock(classOf[BrokerTopicStats]), mockTopicStats1)
 
     try {
-      // make broker 0 the leader of partition 0 and
+      // make broker 1 the leader of partition 0 and
       // make broker 1 the leader of partition 1
```

Comment:
The original [commit](https://github.com/apache/kafka/commit/aa8e850afb7baa447882b493eae29f99a39e35a0) suggests that the comment is actually correct but we changed the leader by mistake afterwards. I suppose that we missed this while [refactoring](https://github.com/apache/kafka/commit/66183f730feaa7dca7b4088053f44015a626a6e8) this code.

Reply:
@dajac Indeed,  should I modify the code according to the  original commit, or modify  the comments according to the current  code ?

Reply:
I would modify the test.

Reply:
@dajac I have modified the test. Thanks for your review.

Reply:
@dajac could you help review the code？Thanks.